app/models/item.rb

```diff
@@ -3,11 +3,12 @@ class Item < ApplicationRecord
   has_many :item_revisions, foreign_key: 'item_uuid', dependent: :destroy
   has_many :revisions, -> { order 'revisions.created_at DESC' }, through: :item_revisions, dependent: :destroy
 
-  after_commit :persist_revision, :cleanup_excessive_revisions
+  after_commit :persist_revision, :cleanup_excessive_revisions, :duplicate_revisions
```

Comment:
We don't want to do the duplicate revisions job after every save do we? The duplicate_of field is permanent once set, so if you save the duplicate item many items, it will run the duplicate job many times. Perhaps take a look at `after_create` instead?

Reply:
good catch, will fix